frame/liquidity-bootstrapping/src/benchmarking.rs

```diff
@@ -0,0 +1,145 @@
+use super::*;
```

Comment:
may consider including benchamrks as tests as it is done in lending. so they break less and increase coverage.